http/rest/handler.go

```diff
@@ -0,0 +1,107 @@
+package rest
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"raccoon/config"
+	"raccoon/logger"
+	"raccoon/metrics"
+	"raccoon/pkg/collection"
+	"raccoon/pkg/deserialization"
+	"raccoon/pkg/identification"
+	pb "raccoon/pkg/proto"
+	"raccoon/pkg/serialization"
+	"time"
+)
+
+type Handler struct{}
+
+func (h *Handler) GetRESTAPIHandler(c collection.Collector) http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		contentType := r.Header.Get("Content-Type")
+		rw.Header().Set("Content-Type", contentType)
+		var group string
```

Comment:
This entire block of code is duplicated across the handlers. Essentially it checks for the group and constructs the group with the identifier. May be this block can move to a util and reused across thr handlers.
WDYT?

Reply:
There is a slight difference, gRPC metadata datatype is `map[string][]string` and for HTTP it's `map[string]string`. So that's why kept it in the handler itself.

Reply:
I also noticed the same. There are a few instrumentations that also seems repeated. Also quite likely will be needed for future handlers. However, I couldn't think of a good way to abstract this. IMO we can delay this change for a while until it's a bit more mature. Similar direction on [this discussion](https://github.com/odpf/raccoon/pull/17#discussion_r740116359).